src/versions.json

```diff
@@ -5,23 +5,33 @@
     "networkId": 22,
     "contracts": {
       "ERC725Account": {
-        "versions": { "0.0.1": "0xe6009BCbC6cb1046E72db3b50Ff0b9F3C94Db2fB" },
+        "versions": {
+          "0.5.0": "0x5EB0Aaac7C7C63F8cB06dbc7F48f9D5E61158E6a"
+        },
         "baseContract": true
       },
       "KeyManager": {
-        "versions": { "0.0.1": "0x36C80542f7ed462cc3fdB2b7AC1a83A80C5dD0f6" },
+        "versions": {
+          "0.5.0": "0x97939C42d23e70b245ce18B20D03191AC451701a"
+        },
         "baseContract": true
       },
       "UniversalReceiverDelegate": {
-        "versions": { "0.0.1": "0xEE495c349eB1ae864bb0e899A71aBD0b660eE4f0" },
+        "versions": {
+          "0.5.0": "0xE2D6038acD92200790Df695Ebd13856CdF2a6942"
+        },
         "baseContract": false
       },
       "LSP7Mintable": {
-        "versions": { "0.0.1": "0x78CFf775e3084E9B064FB59FC25b6143f1ddB831" },
+        "versions": {
+          "0.5.0": "0x99AA90a7C44e5Ce4C1960dc72Dda5C45De37EA4B"
+        },
         "baseContract": true
       },
       "LSP8Mintable": {
-        "versions": { "0.0.1": "0x61ADBa4744d78280d9A0D5012FdA25398068EAFb" },
+        "versions": {
+          "0.5.0": "0x9486f66B2CD54a04B75F8E3b1088C46B02F05F26"
+        },
```

Comment:
We need to add auto CI tool to check these changes for us.

I've checked manually, everything is ok. But next time we better have a CI check to do it for us ;)

Issue created here:
https://github.com/lukso-network/tools-lsp-factory/issues/77